src/datascience-ui/native-editor/nativeCell.tsx

```diff
@@ -448,6 +448,11 @@ export class NativeCell extends React.Component<INativeCellProps> {
         e.stopPropagation();
         e.preventDefault();
 
+        // Refocus the current cell if it is markdown to make it render
+        if (this.isMarkdownCell() && this.wrapperRef && this.wrapperRef.current && this.isFocused()) {
+            this.wrapperRef.current.focus();
```

Comment:
```suggestion
            this.escapeCell(e);
```

Please can we use `escapeCell`, at the end of the day thats what is required. We need to escape the cell. Manually focussing does the exact same thing, but this way its obvious what we are doing and why. With the other approach its unclear. I.e. we know hitting escape will render the markdown, hence we're merely using existing behaviour.

Reply:
ok

Reply:
Also, I would remove the comments as it isn't necessary, at least thats my opinion, cuz to me its a little confusing (cuz we want to focus the cell however `isFocused` is already true for the `nativeCell` - i.e. contradictory).

Reply:
The comment now says: `// Escape the current cell if it is markdown to make it render`